TSP/BLL/Services/RetirePeopleSetadditionalInfo.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace BLL.Services
 {
-    class RetirePeopleSetadditionalInfo : IStrategy
+    public class RetirePeopleSetadditionalInfo : IStrategy
```

Comment:
The name of these classes should end with "*Startegy".

Reply:
fix